### PR TITLE
Keep the sign_up flows in sync with the IdP

### PR DESCRIPTION
**Pull request by jgrevich**
This change keeps this loadtest code in sync with the IdP https://github.com/18F/identity-idp/commit/cdea08fd55ff2f8a1e92a60009dad5b57a491e63

### CODE REVIEW

File: load_testing/common_flows/flow_sp_ial2_sign_up.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def ial2_sign_up(context):
         context,
         "post",
         "/sign_up/create_password",
-        "/two_factor_options",
+        "/authentication_methods_setup",
```
Comment (zachmargolis) [inline load_testing/common_flows/flow_sp_ial2_sign_up.py:106]: cc @18F/identity-katherine just to double check, this URL change is permanent, not part of a feature flag rollout, right?

Comment (mdiarra3) [reply load_testing/common_flows/flow_sp_ial2_sign_up.py:106]: correct, we wanted a clearer url to indicate what is actually happening.

Comment (jgrevich) [reply load_testing/common_flows/flow_sp_ial2_sign_up.py:106]: It sounds like the url change is permanent so we should be good to go, is that right?

**Review by zachmargolis — APPROVED** [review]
LGTM

**Pull request merged**